lib/pod/command/package.rb

```diff
@@ -67,10 +67,13 @@ def build_in_sandbox(platform)
 
         sandbox = install_pod(platform.name)
 
-        perform_build(platform, sandbox)
```

Comment:
I used `Process.exit`, rather than `Process.exit!`, because it works by throwing a `SystemExit` exception which unwinds the stack up to the root -- firing `ensure` blocks and exit handlers along the way -- before terminating the process. So the `ensure` block will be called here, and I've verified that by adding a debug message there.

See http://ruby-doc.org/core-1.9.3/Process.html#method-c-exit